src/test/article.spec.js

```diff
@@ -4,10 +4,12 @@ import sinon from 'sinon';
 import app from '../index';
 import BaseRepository from '../repository/base.repository';
 import { createUser, createArticle, generateArticle } from './utils/helpers';
+import { articleSample } from './mockdata/articledata';
 import db from '../database/models';
 import helper from '../helpers/utils';
 
 const ARTICLES_API = '/api/v1/articles';
+const articleEndpoint = '/api/v1/articles';
```

Comment:
You may want to remove one end point if you're not using the both. Noticed the are the same route.